fdbclient/RYWIterator.h

```diff
@@ -72,6 +74,7 @@ class RYWIterator {
 	SnapshotCache::iterator cache;
 	WriteMap::iterator writes;
 	KeyValueRef temp;
+	bool bypassUnreadable;
```

Comment:
Add a comment:
When set, allows read from sections of keyspace that have become unreadable because of versionstamp operations